src/test/java/com/wikia/webdriver/common/core/CommonExpectedConditions.java

```diff
@@ -62,6 +56,7 @@ public String toString() {
       public Boolean apply(WebDriver from) {
         try {
           String elementsAttributeValue = element.getAttribute(attribute);
+//          PageObjectLogging.log("class in wait", elementsAttributeValue, true);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3ACommentedOutCodeLine)